weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/context.tsx

```diff
@@ -35,3 +36,44 @@ export const WFDataModelAutoProvider: FC<{
     </WFDataModelHooksContext.Provider>
   );
 };
+
+/**
+ * Returns true if the client can connect to trace server and the project has
+ * objects or calls.
+ */
+export const useProjectHasTraceServerData = (
+  entity: string,
+  project: string
+) => {
+  const hasTraceServer = useHasTraceServerClientContext();
+  const objs = tsWFDataModelHooks.useRootObjectVersions(
+    entity,
+    project,
+    {},
+    1,
+    true,
+    {
+      skip: !hasTraceServer,
+    }
+  );
+
+  const calls = tsWFDataModelHooks.useCalls(
```

Comment:
you could optimize this by selecting only the `id` column for example as well

Reply:
done!